gems/sorbet/lib/create-config.rb

```diff
@@ -24,6 +24,8 @@ def self.main
     File.open(SORBET_CONFIG_FILE, 'w') do |f|
       f.puts('--dir')
       f.puts('.')
+      f.puts('--ignore')
+      f.puts('vendor/bundle')
```

Comment:
This ignores `vendor/bundle` anywhere in the path:

```
--ignore string           Ignores input files that contain the given
                          string in their paths (relative to the input
                          path passed to Sorbet). Strings beginning with /
                          match against the prefix of these relative
                          paths; others are substring matchs. Matches must
                          be against whole folder and file names, so
                          `foo` matches `/foo/bar.rb` and `/bar/foo/baz.rb`
                          but not `/foo.rb` or `/foo2/bar.rb`.
```

I think you want

```suggestion
      f.puts('--ignore=/vendor/bundle')
```

Reply:
👍  (though it's slightly inconsistent to have `--dir` on separate lines and `--ignore` as a single line)

Reply:
ok i'm fine any way you prefer

Reply:
:shipit: